### PR TITLE
(DOCSP-39172): Edge Server: Add details about offline demo mode

### PR DESCRIPTION
**Pull request by dacharyc**
## Pull Request Info

Jira ticket: https://jira.mongodb.org/browse/DOCSP-39172

- [Configure Edge Server](https://preview-mongodbdacharyc.gatsbyjs.io/atlas-app-services/DOCSP-39172/edge-server/configure/#simulate-offline-connectivity): New "Simulate Offline Connectivity" section describing how to use offline demo mode.

### Reminder Checklist

Before merging your PR, make sure to check a few things.

- [x] Did you tag pages appropriately?
  - genre
  - programming_language
  - meta.keywords
  - meta.description
- [x] Describe your PR's changes in the Release Notes section
- [x] Create a Jira ticket for related docs-realm work, if any

### Release Notes

- **Edge Server**
  - Configure Edge Server: New "Simulate Offline Connectivity" section describing how to use offline demo mode.

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-app-services/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/edge-server/configure.txt [hunk 1]
```diff
@@ -585,6 +585,77 @@ Where the JSON file contains:
...
+
+When you bring the Edge Server instance out of offline demo mode, it
+automatically resumes syncing with Atlas. You can see the data from the 
+Edge Server instance reflected in Atlas, and data changed in Atlas becomes
+visible to the instance and connected clients.
+
+Set up Offline Demo Mode
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use the following ``edgectl`` flag with either the ``init`` command or the
```
Comment (cbullinger) [inline source/edge-server/configure.txt:611]: does it make sense to spell it out as: you can either create a new instance with the demo or add it to an existing instance?

File: source/edge-server/configure.txt [hunk 2]
```diff
@@ -585,6 +585,77 @@ Where the JSON file contains:
...
+
+.. code-block:: console
+
+   edgectl offline-demo enable-connection
+
+You get a message indicating that the internet connection has been successfully
+enabled. If you check the Edge Server instance status, the ``cloud_connected``
+bool changes from ``false`` to ``true``.
+
+After a moment for any changes to sync, you can see local changes reflected
```
Comment (cbullinger) [inline source/edge-server/configure.txt:656]:
```suggestion
After waiting a moment for any changes to sync, you can see local changes reflected
```
?

File: source/edge-server/configure.txt [hunk 3]
```diff
@@ -585,6 +585,77 @@ Where the JSON file contains:
      "Item": "priority >= 3"
    }
 
+.. _edge-server-simulate-offline-connectivity:
+
+Simulate Offline Connectivity
+-----------------------------
+
+Edge Server provides a demo mode you can use to simulate the instance losing
```
Comment (cbullinger) [inline source/edge-server/configure.txt:593]:
```suggestion
Edge Server provides a demo mode you can use to simulate an Edge Server instance losing
```

**Review by cbullinger — APPROVED** [review]

File: source/edge-server/configure.txt (hunk 1 above)
Comment (dacharyc) [reply source/edge-server/configure.txt:611]: I think people are very unlikely to use it with `init` so I wanted to gloss over that - I think the real use case is `config`. But it's a valid flag for `init` so it seemed like I should at least mention that.

**Pull request merged**